panel/io/reload.py

```diff
@@ -117,48 +179,24 @@ def record_modules():
         except Exception:
             continue
 
-def _reload(module=None):
-    if module is not None:
-        for module in _modules:
-            if module in sys.modules:
-                del sys.modules[module]
-    for cb in _callbacks.values():
-        cb.stop()
-    _callbacks.clear()
+def _reload():
     if state.location is not None:
         # In case session has been cleaned up
         state.location.reload = True
     for loc in state._locations.values():
         loc.reload = True
 
-def _check_file(modify_times, path, module=None):
+def _check_file(modify_times, path):
     try:
         modified = os.stat(path).st_mtime
+    except FileNotFoundError:
+        if path in modify_times:
+            return 3
     except Exception:
-        return
+        return 0
```

Comment:
Can you make a comment or reference to what the different return numbers mean?

Reply:
Will do, effectively this is just roughly mirroring watchfiles codes:

- 0: No change
- 2: File modified
- 3: File deleted